flang/lib/Lower/OpenMP.cpp

```diff
@@ -60,10 +87,16 @@ static void genObjectList(const Fortran::parser::OmpObjectList &objectList,
   }
 }
 
+using privatizeCallbackTy =
```

Comment:
Is it cleanup/leftover code from a different patch ?

Reply:
Nit: leftover still seems to be there.